src/pjpersist/serialize.py

```diff
@@ -192,10 +192,18 @@ def __hash__(self):
         return self.hash
 
     def __eq__(self, other):
-        return self.hash == other.hash
+        try:
+            return self.hash == other.hash
+        except AttributeError:
+            # `other` is not a DBRef or is None
+            return False
 
     def __neq__(self, other):
-        return self.hash != other.hash
+        try:
+            return self.hash != other.hash
+        except AttributeError:
+            # `other` is not a DBRef or is None
+            return True
```

Comment:
Do we need both `__eq__` and `__neq__`?

Reply:
haha good catch, it's `__ne__` not `__neq__`  so it's unused code at the moment

also: For __ne__(), by default it delegates to __eq__() 

https://docs.python.org/3/reference/datamodel.html#object.__ne__